services/api-server/src/simcore_service_api_server/api/dependencies/models_schemas_function_filters.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated, Any
+
+from fastapi import Query
+from models_library.functions import FunctionJobCollectionsListFilters
+from pydantic.fields import FieldInfo
+
+
+def _get_query_params(field: FieldInfo) -> dict[str, Any]:
+    params = {}
+
+    if field.description:
+        params["description"] = field.description
+    if field.examples:
+        params["example"] = next(
+            (example for example in field.examples if "*" in example), field.examples[0]
+        )
+    return params
+
+
+def get_function_job_collections_filters(
```

Comment:
NOTE: I am working in a solution to having to define both the Model and a dependency injection to get the queries with all the examples and doc right into the openapi